importApp/payload

```diff
@@ -0,0 +1,16 @@
+{
+  "commercetools": {
+    "projectKey": "joyridedata-importer-23",
+    "clientId": "86mYf5_30BqPzYBbEiLAY4uM",
+    "clientSecret": "soh3gbYptYdTvUPwF4jy8wNFcM4fFJEJ",
```

Comment:
you checked in your credentials

The payload file should be a json file and in .gitignore